.github/workflows/build.yml

```diff
@@ -81,100 +70,94 @@ jobs:
           bundle install --jobs 4 --retry 3
 
       - name: Check
-        run: bundle exec fastlane check
+        run: bundle exec fastlane test
```

Comment:
This needs to align with the `fastfile` lane names: 
https://github.com/bitwarden/authenticator-android/blob/main/fastlane/Fastfile#L21